### PR TITLE
Update setting-up-snowflake.md

### PR DESCRIPTION
**Pull request by mirnawong1**
added src snippet per instructions here: https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md#reusing-snippets-of-content

need to have pr https://github.com/dbt-labs/docs.getdbt.com/pull/1923 approved first and this is also linked in pr https://github.com/dbt-labs/docs.getdbt.com/pull/959

## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

**Comment by dbeatty10** [comment]
Do you know why the build is failing?

I'm guessing this is the relevant bit in the logs:
```
6:56:25 AM: SyntaxError: /opt/build/repo/website/docs/guides/getting-started/getting-set-up/setting-up-snowflake.md: Expected corresponding JSX closing tag for <br>. (224:877)
```

I wonder if `<br>` within this PR needs to be `<br/>` instead?

Maybe https://github.com/dbt-labs/docs.getdbt.com/pull/1923 needs `<br/>` too?

**Comment by mirnawong1** [comment]
> Do you know why the build is failing?
> 
> I'm guessing this is the relevant bit in the logs:
> 
> ```
> 6:56:25 AM: SyntaxError: /opt/build/repo/website/docs/guides/getting-started/getting-set-up/setting-up-snowflake.md: Expected corresponding JSX closing tag for <br>. (224:877)
> ```
> 
> I wonder if `<br>` within this PR needs to be `<br/>` instead?
> 
> Maybe #1923 needs `<br/>` too?

i'll change that up, thank you @dbeatty10 ! also netlify has downgraded performance here: https://www.netlifystatus.com/ 

slack thread: https://dbt-labs.slack.com/archives/C02NCQ9483C/p1661870355905559

**Comment by dbeatty10** [comment]
The error message is this now:
```
9:01:31 AM: Error: Cannot find module './snowflake-acct-name.md'
```

So like you said, probs need to have https://github.com/dbt-labs/docs.getdbt.com/pull/1923 merged first.

**Comment by mirnawong1** [comment]
thank you v much @dbeatty10 - i have a meeting w Leona tomorrow and will go over this for the merge as it's my first time doing snippets 🙏

File: website/docs/guides/getting-started/getting-set-up/setting-up-snowflake.md [hunk 1]
```diff
@@ -244,7 +244,8 @@ Now let's formally set up the connection between dbt Cloud and Snowflake.
 
 1. Click on the "Snowflake" icon to set up your connection.
 2. Enter the following information under Snowflake settings.
-    * **Account:** Find your account by using the Snowflake trial account URL and removing `snowflakecomputing.com`. The order of your account information will vary by Snowflake version. For example, Snowflake's Classic console URL might look like: `oq65696.west-us-2.azure.snowflakecomputing.com`. The AppUI or Snowsight URL might look more like: `snowflakecomputing.com/west-us-2.azure/oq65696`. In both examples, your account will be: `oq65696.west-us-2.azure`. For more information, see "[Account Identifiers](https://docs.snowflake.com/en/user-guide/admin-account-identifier.html)" in the Snowflake documentation.
+    * **Account:** <Snippet src="snowflake-acct-name"/> <br/>
```
Comment (runleonarun) [inline website/docs/guides/getting-started/getting-set-up/setting-up-snowflake.md]:
```suggestion
    * **Account:** <Snippet src="snowflake-acct-name" /> <br/>
```

File: website/docs/guides/getting-started/getting-set-up/setting-up-snowflake.md [hunk 2]
```diff
@@ -244,7 +244,8 @@ Now let's formally set up the connection between dbt Cloud and Snowflake.
...
+    * **Account:** <Snippet src="snowflake-acct-name" /> <br/>
+    Find your account by using the Snowflake trial account URL and removing `snowflakecomputing.com`. The order of your account information will vary by       Snowflake version. For example, Snowflake's Classic console URL might look like: `oq65696.west-us-2.azure.snowflakecomputing.com`. The AppUI or           Snowsight URL might look more like: `snowflakecomputing.com/west-us-2.azure/oq65696`. In both examples, your account will be: `oq65696.west-us-           2.azure`.    For more information, see "[Account Identifiers](https://docs.snowflake.com/en/user-guide/admin-account-identifier.html)" in the             Snowflake documentation.
```
Comment (dbeatty10) [inline website/docs/guides/getting-started/getting-set-up/setting-up-snowflake.md]: Is there a reason why large spaces were inserted?

Or do we want to remove the large spaces like this?
```suggestion
    Find your account by using the Snowflake trial account URL and removing `snowflakecomputing.com`. The order of your account information will vary by Snowflake version. For example, Snowflake's Classic console URL might look like: `oq65696.west-us-2.azure.snowflakecomputing.com`. The AppUI or Snowsight URL might look more like: `snowflakecomputing.com/west-us-2.azure/oq65696`. In both examples, your account will be: `oq65696.west-us-2.azure`. For more information, see "[Account Identifiers](https://docs.snowflake.com/en/user-guide/admin-account-identifier.html)" in the             Snowflake documentation.  
```

**Review by dbeatty10 — APPROVED** [review]

**Pull request merged**